src/Particle/ParticleContainer.H

```diff
@@ -312,8 +312,34 @@ void make_ParticleContainer_and_Iterators (py::module &m, std::string allocstr)
         //                  const Vector<std::string>& int_comp_names = Vector<std::string>()) const
         // void CheckpointPre ();
         // void CheckpointPost ();
+
         // void Restart (const std::string& dir, const std::string& file);
+        .def("restart",
+            [](
+                ParticleContainerType & pc,
+                std::string const & dir,
+                std::string const & file
+            ){
+                return pc.Restart(dir, file);
+            },
```

Comment:
Does this shortcut work, too?
```suggestion
        .def("restart",
            &ParticleContainerType::Restart,
```